### PR TITLE
feat: COM-3162 update dependencies

### PR DESCRIPTION
**Pull request by ArtyomResh**
## Checklist
- [x] Code meets requirements not only from the description of the task, but also from its comments
- [x] Code works correctly on all resolutions
- [x] Task completed with comments for testing (if necessary, otherwise this item can be removed)
- [ ] Browser dev tools has no errors

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```
Comment (maksim-kononov-csssr) [inline .husky/pre-commit:2]: Это сам Хаски создал?

File: .husky/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+_
```
Comment (maksim-kononov-csssr) [inline .husky/.gitignore:1]: И вот этот файл, это сам Хаски создал?

**Review by maksim-kononov-csssr — APPROVED** [review]

File: .husky/pre-commit (hunk 1 above)
Comment (ArtyomResh) [reply .husky/pre-commit:2]: Да

File: .husky/.gitignore (hunk 2 above)
Comment (ArtyomResh) [reply .husky/.gitignore:1]: Угу

**Pull request merged**